themes/default/css/_leftNav.scss

```diff
@@ -1,18 +1,13 @@
 #leftNav {
 	background: $grey-dark;
-
-	width: 100%;
 	padding: 0;
 	position: relative;
-
-	width: 35%;
-	position: absolute;
-	top: 0;
-	left: 0;
-	bottom: 0;
+	z-index: 2;
+	float: left;
+	height: auto;
 
 	@include breakpoint(baby-bear) {
-		
+		width: 35%;
```

Comment:
Yeah I think you're right but I think the problem is actually in the definitions of the breakpoints.